src/views/GeneralReception.vue

```diff
@@ -1,159 +1,157 @@
 <template>
-  <flagged-feature name="dpl_277_enable_general_reception">
-    <div class="flex flex-row space-x-12 px-12">
-      <loading-full-screen-modal v-bind="modalState"></loading-full-screen-modal>
-      <div class="w-1/2 space-y-8">
-        <div>
-          <traction-heading level="4" :show-border="true"> Source </traction-heading>
-          <traction-muted-text class="float-left"
-            >The location to import the labware from
-          </traction-muted-text>
-          <traction-select
-            id="sourceSelect"
-            v-model="source"
-            class="inline-block w-full"
-            :options="receptions"
-            data-type="source-list"
-          />
-        </div>
+  <div class="flex flex-row space-x-12 px-12">
```

Comment:
The file changes look ugly but I the only difference is the removal of the flagged-feature wrapper and then subsequent indentation changes.